pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUserStudyEnrollment.java

```diff
@@ -33,6 +33,17 @@
     @CreateSqlObject
     ActivityInstanceDao getActivityInstanceDao();
 
+    @SqlQuery("select user_id"
+            + "  from user_study_enrollment"
+            + " where study_id = :studyId"
+            + "   and valid_to is null"
+            + " order by user_id"
+            + " limit :limit offset :offset")
```

Comment:
Here again, it probably the enrollment start time (or if not indexed, perhaps better the user_study_enrollment_id) might be a better order key.